### PR TITLE
Added Contactpage and modilied index.html

### PR DESCRIPTION
**Pull request by Federicofilini**

### CODE REVIEW

File: contact-us.html [hunk 1]
```diff
@@ -0,0 +1,573 @@
...
+
+            Connect
+          </button>
+        </div>
+      </div>
+    </header>
+
+    <section class="blog_section">
+      <div class="header-nav_block">
+        <h1 class="blog_header">Our blog</h1>
```
Comment (andacenko) [inline contact-us.html:173]: Change text on "Contact Us"

File: contact-us.html [hunk 2]
```diff
@@ -0,0 +1,573 @@
...
+          <li class="list__item__bar">
+            <a href="catalog_section" class="header_nav_link">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="search_box">
+        <div class="search__field">
+          <button class="search-btn">
+            <svg
```
Comment (andacenko) [inline contact-us.html:110]: 1. Very small svg for search.
Should be width="24" height="24"
2. Add styles for all svg (remove bottom indent):
```
svg {
          vertical-align: top;
   }
```

File: contact-us.html [hunk 3]
```diff
@@ -0,0 +1,573 @@
...
+          </li>
+          <li class="list__item__bar">
+            <a href="catalog_section" class="header_nav_link">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="search_box">
+        <div class="search__field">
+          <button class="search-btn">
```
Comment (andacenko) [inline contact-us.html:109]: Add type="submit"
In the most cases all forms should have submit button to send request.

File: index.css [hunk 4]
```diff
@@ -81,19 +84,17 @@ body {
...
 }
 
 .search-btn {
-  width: 40px;
-  height: 40px;
   padding: 10px;
   background-color: transparent;
   cursor: pointer;
 }
 .search-input {
```
Comment (andacenko) [inline index.css:96]: Please delete for the .search-input
```
flex-grow: 1;
background-image: url("your-custom-clear-icon.svg"); 
```

File: contact-us.html [hunk 5]
```diff
@@ -0,0 +1,573 @@
...
+            <a href="catalog_section" class="header_nav_link">Catalog</a>
+          </li>
+          <li class="list__item__bar">
+            <a href="catalog_section" class="header_nav_link">Contact Us</a>
+          </li>
+        </ul>
+      </nav>
+
+      <div class="search_box">
+        <div class="search__field">
```
Comment (andacenko) [inline contact-us.html:108]: search__field this is should be tag `<form>`

File: contact-us.html [hunk 6]
```diff
@@ -0,0 +1,573 @@
...
+        <ul class="blog-page_breadcrumbs">
+          <li><a class="blog-page_link" href="">Home</a></li>
+
+          <li>Contact Us</li>
+        </ul>
+      </div>
+    </section>
+    <section>
+      <!-- ------------------------------------------ -->
+      <ul class="header-nav-info_list">
```
Comment (andacenko) [inline contact-us.html:183]: If you use `<ul>` then should be `<li>` tags inside.
Please add `<li>`.

File: contact-us.css [hunk 7]
```diff
@@ -0,0 +1,297 @@
...
+}
+
+.header-nav-info_list {
+  display: flex;
+  justify-content: center;
+  margin: 64px 0px;
+  gap: 16px;
+}
+.header-nav-info_linc {
+  width: 328px;
```
Comment (andacenko) [inline contact-us.css:14]: Don't use fixed width

File: contact-us.css [hunk 8]
```diff
@@ -0,0 +1,297 @@
...
+  display: flex;
+  align-items: center;
+  border: 2px solid rgba(208, 215, 251, 0.17);
+  flex-direction: column;
+  padding: 40px 32px;
+  gap: 24px;
+  color: #f8f7ff;
+  font-family: Bakbak One;
```
Comment (andacenko) [inline contact-us.css:22]: Please use whole font-family list.

File: contact-us.css [hunk 9]
```diff
@@ -0,0 +1,297 @@
...
+  margin-bottom: 40px;
+}
+.form {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  gap: 24px;
+}
+.form_header-block {
+  width: 640px;
```
Comment (andacenko) [inline contact-us.css:72]: 1. Replace `width: 100%;`
2. Add `box-sizing:border-box;`

File: contact-us.html [hunk 10]
```diff
@@ -0,0 +1,573 @@
...
+              type="checkbox"
+              id="use-profile-info"
+              required
+            />
+            <span class="slider"></span>
+          </label>
+        </div>
+
+        <div class="form_input-box">
+          <p>Name</p>
```
Comment (andacenko) [inline contact-us.html:272]: Delete all this `<p>` use `<label>` instead

File: contact-us.html [hunk 11]
```diff
@@ -0,0 +1,573 @@
...
+          <label class="is-none" for="name">Full Name:</label>
```
Comment (andacenko) [inline contact-us.html:273]: Remove `class="is-none"` 
Add styles for label

File: contact-us.css [hunk 12]
```diff
@@ -0,0 +1,297 @@
...
+/* ------------------------------------------------------------------------------------------ */
+.form_input-box {
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+}
+
+.input-stile {
+  width: 100%;
```
Comment (andacenko) [inline contact-us.css:136]: Delete  width:100%. We have the same one in .contact-us_main input

File: contact-us.css [hunk 13]
```diff
@@ -0,0 +1,297 @@
...
+}
+
+.input-stile:focus {
+  border: 2px solid #3892f3;
+
+  outline: none;
+}
+/* .input-stile:invalid {
+  border: 2px solid  #EA3829
+} */
```
Comment (andacenko) [inline contact-us.css:154]: Please uncomment this block and use :user-invalid pseudo class instead :invalid

File: contact-us.css [hunk 14]
```diff
@@ -0,0 +1,297 @@
...
+  line-height: 20px;
+  margin: 0 10px 6px 0;
+}
+
+/* ---------------------------------Button--------- */
+.radio_button-box {
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 20px;
+  color: #f5f8fb;
```
Comment (andacenko) [inline contact-us.css:187]: Delete color property

File: contact-us.html [hunk 15]
```diff
@@ -0,0 +1,573 @@
...
+
+          <div class="radio_button-box">
+            <div>
+              <input
+                class="radio-input is-none"
+                type="radio"
+                id="male"
+                name="Gender"
+                value="male"
+                required
```
Comment (andacenko) [inline contact-us.html:308]: Add `checked` attribute here

File: contact-us.html [hunk 16]
```diff
@@ -0,0 +1,573 @@
...
+            required
+          />
+        </div>
+
+        <div class="form_input-box">
+          <p>Phone Number</p>
+          <label class="is-none" for="your-phone-number">Phone Number:</label>
+          <input
+            class="input-stile"
+            type="number"
```
Comment (andacenko) [inline contact-us.html:344]: This is `type="tel"`. Please change.

File: contact-us.html [hunk 17]
```diff
@@ -0,0 +1,573 @@
...
+            id="your-phone-number"
+            name="Phone number"
+            required
+          />
+
+          >
```
Comment (andacenko) [inline contact-us.html:350]: Remove this `>` symbol

File: contact-us.css [hunk 18]
```diff
@@ -0,0 +1,297 @@
...
+}
+/* .input-stile:invalid {
+  border: 2px solid  #EA3829
+} */
+
+.form_input-box textarea {
+  background-color: #0d0d15;
+  border: 2px solid rgba(208, 215, 251, 0.17);
+  color: #f5f8fb;
+  padding: 10px;
```
Comment (andacenko) [inline contact-us.css:160]: Add resize: none;

File: contact-us.css [hunk 19]
```diff
@@ -0,0 +1,297 @@
...
+.form_input-box textarea:focus {
+  border: 2px solid #3892f3;
+  outline: none;
+  caret-color: #0265dc;
+}
+
+.form_input-box textarea:hover {
+  border: 2px solid #3892f3;
+  outline: none;
+}
```
Comment (andacenko) [inline contact-us.css:171]: Add :user-invalid styles here for textarea

File: contact-us.html [hunk 20]
```diff
@@ -0,0 +1,573 @@
...
+                    height="24"
+                /></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+      </div>
+    </footer>
+  </body>
+</html>
```
Comment (andacenko) [inline contact-us.html:573]: Please add Catalog page

File: contact-us.html (hunk 1 above)
Comment (Federicofilini) [reply contact-us.html:173]: Done )))

File: contact-us.html (hunk 2 above)
Comment (Federicofilini) [reply contact-us.html:110]: Done

File: contact-us.html (hunk 3 above)
Comment (Federicofilini) [reply contact-us.html:109]: Done

File: index.css (hunk 4 above)
Comment (Federicofilini) [reply index.css:96]: Done

File: contact-us.html (hunk 6 above)
Comment (Federicofilini) [reply contact-us.html:183]: Done

File: contact-us.css (hunk 8 above)
Comment (Federicofilini) [reply contact-us.css:22]: Done

File: contact-us.css (hunk 9 above)
Comment (Federicofilini) [reply contact-us.css:72]: Done

File: contact-us.html (hunk 10 above)
Comment (Federicofilini) [reply contact-us.html:272]: Done

File: contact-us.css (hunk 12 above)
Comment (Federicofilini) [reply contact-us.css:136]: Done

File: contact-us.css (hunk 13 above)
Comment (Federicofilini) [reply contact-us.css:154]: Done

File: contact-us.css (hunk 14 above)
Comment (Federicofilini) [reply contact-us.css:187]: Done

File: contact-us.html (hunk 15 above)
Comment (Federicofilini) [reply contact-us.html:308]: Done

File: contact-us.html (hunk 17 above)
Comment (Federicofilini) [reply contact-us.html:350]: Done

File: contact-us.css (hunk 18 above)
Comment (Federicofilini) [reply contact-us.css:160]: Done

File: contact-us.css (hunk 19 above)
Comment (Federicofilini) [reply contact-us.css:171]: Done

File: contact-us.html (hunk 20 above)
Comment (Federicofilini) [reply contact-us.html:573]: Done

File: contact-us.css (hunk 7 above)
Comment (Federicofilini) [reply contact-us.css:14]: Done

File: contact-us.html (hunk 5 above)
Comment (Federicofilini) [reply contact-us.html:108]: Done

File: contact-us.html (hunk 1 above)
Comment (andacenko) [reply contact-us.html:173]: Still exist. Should be Contact us

Screenshot 2024-12-12 at 11 07 19

**Pull request merged**